.travis.yml

```diff
@@ -1,17 +1,26 @@
 language: python
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+  #- "2.7"
+  #- "3.4"
+  #- "3.5"
+  #- "3.6"
+  - "3.7"
+# Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
 cache:
   directories:
     - /home/travis/virtualenv/python2.7/lib/python2.7/site-packages
     - /home/travis/virtualenv/python3.4/lib/python3.4/site-packages
     - /home/travis/virtualenv/python3.5/lib/python3.5/site-packages
     - /home/travis/virtualenv/python3.6/lib/python3.6/site-packages
+    - /home/travis/virtualenv/python3.7/lib/python3.7/site-packages
 install:
-  - pip install -r requirements.txt
+  - sudo apt install python3-lxml python3-pil python3-reportlab
```

Comment:
It is suggested to not use `apt` but `pip` for installing python packages: https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs